### DISCUSSION TITLE
Haier Hon

### DISCUSSION THREAD

**Discussion by quickgold1** [indomus/forum#3111] [HUB: Home Assistant]
Ciao a tutti
Oggi ho effettuato l'aggiornamento del core 2024.6.0 di Home Assistant,  e le entità "climate" del mio Condizionatore Haier,
hanno smesso di essere comandate, vengono rilevati solo i sensori.
Qualcuno sà se per caso è un mio problema di configurazione,
oppure è proprio Haier che non è più possibile integrarla in HA?
Grazie

**Comment by Scialla** [comment]
In ogni caso, presumo tu utilizzi qualche integrazione, ed a quella devi fare riferimento leggendoti le issue su github

Se usi Haier hOn le issue le trovi qui:
https://github.com/Andre0512/hon/issues

probabilmente la prima è già di tuo interesse, seguila. Se non è il tuo caso (lo puoi sapere solo tu guardando i log) aprine una

**Reply by quickgold1** [reply]
Si grazie, in quella issue si descrive proprio il problema che ho.
Quindi devo desumerne che  l'integrazione non è più autorizzata a controllare il Climatizzatore.

**Reply by Pedrone76** [reply]
Ciao, hai risolto in qualche modo?
